docs/DeveloperGuide.md

```diff
@@ -307,31 +310,112 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `ArtBuddy`
+and the **Actor** is the `user`, unless specified otherwise)
+
+**Use case: Delete a customer**
+
+**MSS**
+
+1. ArtBuddy shows a list of customers
+2. User requests to delete a specific customer in the list
+3. ArtBuddy deletes the customer and all the customer's commissions
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. The customer list is empty.
+
+  Use case ends.
+
+* 2a. The given index is invalid.
+
+    * 2a1. ArtBuddy shows an error message.
+
+      Use case resumes at step 2.
```

Comment:
Hm, should the use case resume after showing an error message because there's no valid index for AB to show?

Reply:
Yap u make sense, I will change it